tests/integration/targets/setup_docker_registry/tasks/setup.yml

```
@@ -100,7 +106,78 @@
   - set_fact:
       registry_frontend_address: 'n/a'
     when: can_copy_files is failed
-  - debug: msg="Registry available under {{ registry_address }}, NGINX frontend available under {{ registry_frontend_address }}"
+
+  # Set up second nginx frontend for registry
+  - name: Start nginx frontend for registry
+    docker_volume:
+      name: '{{ docker_registry_container_name_nginx2 }}'
+      state: present
+  - name: Create container for nginx frontend for registry
```

```suggestion
      state: present
 
  - name: Create container for nginx frontend for registry
```
listed in best practices. this would make the tests more readable

---

It doesn't make sense to only change a small part of this setup role to the new style, and keep all the rest. Everything should be changed at once in a separate PR.

---

definitely should be

---

See #1161.